### DISCUSSION TITLE
Timelapse option

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#5229] [Ideas]
Would it be possible to add an option to create a timelapse video?

 inputs:
        - path: rtsp://192.168.0.224:554/bla/bla
                roles:
            - detect
            - record
            _**- timelapse         
               - options
                    framerate: 
                    etc... etc...**_

**Comment by NickM-27** [comment]
I don't think there's a reason to have a separate role for timelapse, they can definitely be created from the existing recording files. In any case, there is already a feature request for this https://github.com/blakeblackshear/frigate/issues/609

**Reply by 18923** [reply]
Thank you, my search argument did not find that entry #609
Hope it will be implemented some day.
